src/utils/Sort.js

```diff
@@ -647,13 +649,26 @@ export function cardGetLabels(card, sort) {
     return [];
   } else if (sort == 'Creature/Non-Creature') {
     return typeLine(card).toLowerCase().includes('creature') ? ['Creature'] : ['Non-Creature'];
-  } else if (sort == 'Price') {
-    var price = null;
-    if (card.details.price) {
-      price = card.details.price;
-    } else if (card.details.price_foil) {
-      price = card.details.price_foil;
+  } else if (sort == 'Price USD') {
+    var price = card.details.prices.usd || card.details.prices.usd_foil;
+    if (price) {
```

Comment:
this could be moved to a function to be repeated for each price type?

Reply:
yes